barretenberg/cpp/src/barretenberg/stdlib/recursion/honk/verifier/protogalaxy_recursive_verifier.cpp

```diff
@@ -1,69 +1,9 @@
 #include "protogalaxy_recursive_verifier.hpp"
 #include "barretenberg/polynomials/polynomial.hpp"
 #include "barretenberg/proof_system/library/grand_product_delta.hpp"
+#include "barretenberg/stdlib/recursion/honk/verifier/recursive_instances.hpp"
 namespace bb::stdlib::recursion::honk {
 
-template <class VerifierInstances>
```

Comment:
Where did this go?

Reply:
The verifier accumulator (\phi*) is received as input (line 39 in the header file) and it's the verifier output from the previous round of folding. Hence, there is no need for the prover to send it via the transcript. In fact, the prover doesn't even have the complete verifier accumulator since we don't compute the VK on prover side anymore